src/session/config.ts

```diff
@@ -0,0 +1,12 @@
+const getEnvironmentValue = (key: string, defaultValue?: any): string => {
```

Comment:
we already have a file for this - properties.ts

Reply:
Moved this.